test/paper-dialog-scrollable.html

```diff
@@ -86,63 +86,81 @@
 
     <script>
 
+      function runAfterScroll(node, scrollTop, callback) {
```

Comment:
Sure, I'll make a note to do it: https://github.com/PolymerElements/paper-dialog-scrollable/issues/7